src/Models/AZ3166Device.ts

```diff
@@ -762,11 +762,12 @@ export class AZ3166Device extends ArduinoDeviceBase {
   }
 
   private async getComList(): Promise<SerialPortInfo[]> {
-    const lists = await AZ3166Device.serialport.list();
-    if (!lists.length) {
-      vscode.window.showInformationMessage("No serial port is available.");
+    try {

```

Comment:
If we do nothing with err, can we remove the wrapper of try...catch?

Reply:
fixed.